### ISSUE TITLE
DAILY UPDATE AT DAY-END

### ISSUE THREAD

**Issue by dhruvin-restalgo** [RestAlgo/TradingApplication#210]
Enter Daily Updates with basic bullet points.

e.g.

- Developed : user management in Admin login
- Developing in progress : risk form design, limit management form etc.
- Read book on sql
- Took refence for designing from xyz place.

**Comment by VIRAG-RestAlgo** [comment]
* Live : Working in live deployment TBT and exe's
* Live : Working in live testing version with TBT testing with Lokeshsir and Vaibhavbhai.
* Algo Engine : Config file update as per new concept (Done)
* NSEFO Delegate : Config file update as per new concept (WIP)

**Comment by VIRAG-RestAlgo** [comment]
Live : deployment exe's
Live : Working in live testing version with TBT testing with Lokeshsir and Vaibhavbhai.
NSEFO Delegate : Config file update as per new concept (Done)

**Development**
- GR connection is in delegate resolved with lokeshsir.
- Algoengine : Any leg max qty handle with any order request from algoengine.
- Engine : Order main qty handle with min order qty in order book
- Engine : On order reject pending qty handle with final volumnefill qty.

**Comment by VIRAG-RestAlgo** [comment]
Development : Fix Bug for re login client application.

Planning for yesterday's live issues and assigning tasks.

**Comment by VIRAG-RestAlgo** [comment]
**Bug Fix (In Live Dated. 14-06-2024) in Trading Application [Task ID : 241]:-** 

- Net lot should be less then or equal to Execute Lot (Forward and Reverse)
- Clean rule issue while re active rule, forward cycle not clean.
- Try to deactive rule on FNetLot & RNetLot columns is not working.
- If user logout and login screen show at that time user can't re-login properly.
- New : Shfit + F1 to cancel all
- FNetLot 10 and FExecute Lot 12 show.
- Order book open from market depth.
- DPR value 0.05 - 15.50, if price enter 0.05 or 15.50 then low message should not be show.
- By default order lot from GSet and FS prices and RS price show 1st time in F1 and F2.
- If rule active and press double click then open that rule in rule form.

**Comment by VIRAG-RestAlgo** [comment]
1. Live testing with Traders, Sir, and IT & Support team till market off

Bug fix in trading application :

1. Normal new buy and sell form order qty show as minimum lot size.
2. Market depth close button hide.
3. OnClose disconnect message show on top.

**Comment by VIRAG-RestAlgo** [comment]
Live : Testing with trader.

Development : NSEFO re-connect with actual gateway then it should be connected message show to client application.
Development : Non driver leg modification qty mismatch (updated in live and tested in live and working Ok.)

Planning for new task added and Development task merging.

**Comment by VIRAG-RestAlgo** [comment]
Live : Testing with trader.

Development : 1 lot max received in today live testing, development and tested with exchange simulation.

Work with Ushaben and Anilbhai for particular task.

**Comment by VIRAG-RestAlgo** [comment]
Live : Testing with trader.

Code Review for file downloader with new concept.

**Comment by VIRAG-RestAlgo** [comment]
Dev. : 

1. Live FS RS and Save FS RS required in different - different location like F1 / +
2. Last tick save logic changed,.
3. File Download code re-structure with APN due to some issue.

Build new dll : TA, TAAdmin, Engine. for next Monday

**Comment by VIRAG-RestAlgo** [comment]
Try to connect CM in test market.
Dhruvinbhai ID added and test in live market.
CM Order modification.

**Comment by VIRAG-RestAlgo** [comment]
Live : Trading System connectivity testing.

Dev. : CM Order Modification

**Comment by VIRAG-RestAlgo** [comment]
CM Delegate changes for config form and delegate main view.

**Comment by VIRAG-RestAlgo** [comment]
Live : TBT testing
CM Modification and Message Download.

**Comment by VIRAG-RestAlgo** [comment]
Live : TBT data listen from WListen.
Dev. : Message download packet - Done.
Dev. : CM Order RL Modification - Running.
Dev. : CM Order RL cancel - Done.
Neat+ : Decode order confirmation and order modification which order sent from neat, decode in our system message download.
Session : Async await with team. (Diff. diff method)

**Comment by VIRAG-RestAlgo** [comment]
1. NSECM Order Modification with Exchange support team.
2. Dev : Login form title update and screen shot given to support team for approval query.
3. Dev : if ratio is 0 then not wait for order in spinner
4. Dev Merge : NSEFO validation with KAR
5. Dev Merge : Dock style user wise with UPM
6. Dev : NSECM find tuning for stream id as discussed with NSE help desk.

**Comment by VIRAG-RestAlgo** [comment]
Dev Merged : Index average tool merged.
Dev test : Index view.

**Comment by VIRAG-RestAlgo** [comment]
Dev. and test for index view and index average done.

**Comment by VIRAG-RestAlgo** [comment]
1. Live : Index live testing.
2. Index view changes task allot to Anilbhai
3. Dev. NSEFO if delegate disconnected with actual gateway then message pass to client.
4. Dev. NSEFO if notification save in jsong
5. Dev. NSECM order modification done with Lokeshsir.

**Comment by VIRAG-RestAlgo** [comment]
Live : with Delhi client issue
Dev. : Disparity dev. done testing and discussion with Vaibhavbhai , Dipkabhai and sir.
Dev. non driver leg qty not trade and next cycle is run. (Delhi client issue today) - this task WIP

**Comment by VIRAG-RestAlgo** [comment]
Dev. non driver leg qty not trade and next cycle is run. (Delhi client issue yesterday) - this task WIP (with sir)

**Comment by VIRAG-RestAlgo** [comment]
Dev. non driver leg qty not trade and next cycle is run. (Delhi client issue yesterday) - this task WIP (with sir)

**Comment by VIRAG-RestAlgo** [comment]
Spinner Change test in yesterday recording.
Dev & merged : if any rejection or disconnect delegate then disconnect message sent till client.
Dev : Duplicate rule no and merged in live branch.
Spinner optimization option find.

**Comment by VIRAG-RestAlgo** [comment]
Dev and testing with tester for algo engine issue.

Dev. : Simulation duplicate trade issue resolved.

**Comment by VIRAG-RestAlgo** [comment]
Date : 16.07.2024

Algo testing and fix bug

**Comment by VIRAG-RestAlgo** [comment]
Dev. and testing in algo spinner issue regarding next cycle running and other leg not execute with sir.

**Comment by VIRAG-RestAlgo** [comment]
Dev and testing : algo spinner and Ok for qty issue and cycle issue.
Dev. : in trading , spinner and algo fs rs calculation for butterfly strategy.
merged : indices avg. if strike price and ce pe token not added then automatically add with KAR.
merged : file path
colo server : colon copy from live to IT testing db and API with APN.

**Comment by VIRAG-RestAlgo** [comment]
Date : 19-07-2024

Working in live issue like qty and some time order cycle not modify

**Comment by VIRAG-RestAlgo** [comment]
last Friday bug fix in spinner and test with 5 rule and NetQty 100,200,300,500,1000 and order lot 1.
issue solved and deploy in live and support exe

**Comment by VIRAG-RestAlgo** [comment]
Date 22-23-24 July 2024

Algo logic update with Lokeshsir, Vaibhavbhai and Dhruvinbhai and testing with Dipakbhai.

**Comment by VIRAG-RestAlgo** [comment]
Live and Dev : as per today live demo compare with ATINS and after communication with Vaibhavbhai and sir changes in driver and nondriver leg price. changes done and deploy in live.

**Comment by VIRAG-RestAlgo** [comment]
Live and Dev : Algo logic change and domo of client software.
Office shifting.
New logic for driver is start as per new broadcast

**Comment by VIRAG-RestAlgo** [comment]
Dev :Spinner new logic apply driver leg send or modify on every broadcast.
Deploy in live

**Comment by VIRAG-RestAlgo** [comment]
Dev and Live Test : AlgoSpinner for nondriver leg waiting time and cycle execution,. issue solved and test with Dipakbhai with all type of strike with Lokeshsir and it is ok work.
Code merged : algo rule update popup, Trading system memory optimization, Gset driver leg max order qty.

**Comment by VIRAG-RestAlgo** [comment]
1. Algo testing with Dipakbhai in live
2. Code merged with Anilbhai for tradebok reverse fill.
3. Code merged with Pratikbhai export to csv in trading application.
4. Code check with Ushaben for task id 24
5. Code check and test with KAR for task id 356 and 354. (final testing pending for above 5 lacs orders)
6. AlgoRow F1 entry from grid with Ushamadam

Sr. No 2 and 3 deploy in live.

**Comment by VIRAG-RestAlgo** [comment]
Date 01-08-2024
Dev and Live : trading system and update in live exe.

**Comment by VIRAG-RestAlgo** [comment]
In live testing issue check and try to understand what is problem in spinner and create market watch expire token change with new token.

**Comment by VIRAG-RestAlgo** [comment]
Date : 03 Aug 2024

1. Mock in RIKHAV with Anilbhai
2. Strategy master flow merged and test in live. (Admin to engine and engine to delegate) also given demo to support team.
3. work with DJK for NSECM.

**Comment by VIRAG-RestAlgo** [comment]
1. Live testing with Dipakbhai.
2. Work with DJK for NSECM
3. Discussion with sir for event
4. Dev. in spinner regarding cycle complete + stop FSRS calculation test in bvn server and deploy in live will test on 06aug.

**Comment by VIRAG-RestAlgo** [comment]
Dev : Also spiner with sir for live issue all cycle complete then new order sent.
Dev and bug fix : Task ID 381 and live deploy
Dev and testing NSECM login done in test market Ctcl ID 25801

**Comment by VIRAG-RestAlgo** [comment]
1) Live testing with dipakbhai, sir dhruvinbhai for disparity
2) Dev. Spinner : after cancel order wait for confirmation
3) Dev. and merge : Task ID 377,385,386

**Comment by VIRAG-RestAlgo** [comment]
Date : 08-08-2024
Live Testing with Sir and new correction in spinner till 1.00 AM.
and other work with team.

**Comment by VIRAG-RestAlgo** [comment]
Live testing and dev in spinner with dipakbhai and sir.
net position export code merge and deploy in live.
NSECm modification in process

**Comment by VIRAG-RestAlgo** [comment]
Date : 10-08-2024

NSECM : RL new, modification and cancel order packet clear and SL new order WIP.

New Logic merge in Trading application and NSEFO delegate and test with our simulation and exchange simulation (exchange simulation is not test because order is reject due to trading session is not start) 

New logic is deploy in live.

**Comment by VIRAG-RestAlgo** [comment]
1. Live testing with last Saturday deployment.
2. NSECM SL new and modify order request sent and confirmation proper received.

**Comment by VIRAG-RestAlgo** [comment]
Live testing.
In Live ops engine shutdown check with PPB.
NSECM SL order new modify and cancel done only after SL trigger cancel is pending.

**Comment by VIRAG-RestAlgo** [comment]
NSECM code merge and check
Live testing with support team 2 hr
Discussion for Reverse driver leg
AlgoID 0 error check with Log and resolved and deploy new exe

**Comment by VIRAG-RestAlgo** [comment]
1. Live testing with trading team approx. 1.5 hr.
2. NSECM Trimmed Order testing in test market.
3. NSECM order all packet in out testing in test market.
4. Developing in progress : Start working in partial cycle reverse.
5. Log check with UPM for strategy blank in two app id 102 and 1089 in 21-08-2024.

**Comment by VIRAG-RestAlgo** [comment]
1. Developing in progress : Start working in partial cycle reverse.
2. Json file error log resolved.
3. On Grid changes price issue resolved and live update in live market time today.

**Comment by VIRAG-RestAlgo** [comment]
Developing and Discussion in progress : Start working in partial cycle reverse.
NXBlock Algo Design merge with UPM 90 % Done.
RuleWise Net position cross red line issue check and resolved.
Possibilities and required for DriverSqure off note down and send to sir.
FExLot and RExLot error of collection changed, this issue discuss with sir and start new optimized logic.

**Comment by VIRAG-RestAlgo** [comment]
Bug Fixing 81 task review and comment 
1. NX Block Algo watch merge in NX Block Develop branch and tested in BVN server.
2. SqureOff drive leg logic discussed with sir

**Comment by VIRAG-RestAlgo** [comment]
Dev : NSECM contract load and token add in MW and test NSECM broadcast.
Dev and Merge in NSECM 414 and 418
Dev and correct error in TA : TaskID 411 (rule wise net position and F8 issue in trade book first time in day)

**Comment by VIRAG-RestAlgo** [comment]
Dev : AlgoSpinner reverse driver leg order
NSECM : Check with DJK login , NSECM Contract load and strategy master load, user list load.
Work with Dipakbhai almost 1 hr.

**Comment by VIRAG-RestAlgo** [comment]
My task summary from 02 sep to 04 sep

1. Work in NSECM indicator with APN.
2. Con Rev order processing.
3. Ration not decreasing properly issue is in process.
4. Deployment process with Abhishekbhai, Gauravbhai and Ravibhai for static ip issue.
5. Demo to Vaibhavbhai on date 02 sep for NSECM indicator
6. Meeting with Dhruvinbhai on date 02 sep for Delhi client design and new two repots (reports discussion pending and it will discuss on date 05-09-2024)

**Comment by VIRAG-RestAlgo** [comment]
1. TaskID 456
2. Work with APN for CM Indicator
3. Work for Execute lot problem but not solve yet and discussed with Dhruvinbhai also. this logic will change and new logic will discussed on date 06-09-2024 with Dhruvinbhai.

**Comment by VIRAG-RestAlgo** [comment]
Date : 06 - 09- 2024 

NSEFo broadcast merge in SID application.
Box Strategy FS RS logic discuss with Dhruvinbhia.
Abhishek design and code merge and check

**Comment by VIRAG-RestAlgo** [comment]
Abhishek design for index and algo market watch Execute lot and BRK add and deploy.

**Comment by VIRAG-RestAlgo** [comment]
Work in below task 
410,411,443,447,366,
468,470,471
464,469 

and deploy today in live.  work with UPM, DJK, SKP, ARV, KAR

**Comment by VIRAG-RestAlgo** [comment]
TaskID 471, 474, FExLot and RExLot correction for conRev

**Comment by VIRAG-RestAlgo** [comment]
Task ID 474 , Changes check in engine and our simulation
Next planning discussion with Dhruvinbhai, Vaibhavbhai, Lokeshsir.

**Comment by VIRAG-RestAlgo** [comment]
Task ID 474(TA) and 478 (NSECM)

**Comment by VIRAG-RestAlgo** [comment]
Dev. Box Strategy FS RS calculation, 
NSECM - Partial order modification.

**Comment by VIRAG-RestAlgo** [comment]
1. NSECM Order Modification partial test.
2. Box Strategy order processing WIP.

**Comment by VIRAG-RestAlgo** [comment]
1. Trading System
	Task ID : 476 (Box Strategy Added)
 
2. Algo Spinner 
	Task ID : 481,480 (Box Strategy Added)

Development done and deployed

**Comment by VIRAG-RestAlgo** [comment]
Box Order processing start

**Comment by VIRAG-RestAlgo** [comment]
Date 23 and 24 Sep
TaskID 494,484,495,496, 190, 491 (In some task test and merge and some task dev by me)

**Comment by VIRAG-RestAlgo** [comment]
Date 26 Sep 2024

1. IOC Order processing.
2. NSECM Session handle and apply SeqNoVsAppOrderID for which response received from exch.

**Comment by VIRAG-RestAlgo** [comment]
Today,

1. IOC Order Processing Logic done, testing pending will tomorrow in our simulator.
2. Start Stradle FSRS logic.

**Comment by VIRAG-RestAlgo** [comment]
1. Simulation update as per IOC strategy with PPB
2. IOC testing and correction in squreOff logic.

**Comment by VIRAG-RestAlgo** [comment]
Stradel rule new add in AMW with UPM and ARV
IOC Order processing WIP

**Comment by VIRAG-RestAlgo** [comment]
Date 03, 04 and 05

* Straddle and Strangle FSRS and Order Processing.
* IOC Order Processing Done and only pending Execute lot reduce or maintain properly.
* Algo Rule strategy wise code merge with UPM and ARV.

**Comment by VIRAG-RestAlgo** [comment]
Date 07-10-2024

1. PriceDiff_IOC and ConRev_IOC both strategy rule create and add in algo market watch.
2.

**Comment by VIRAG-RestAlgo** [comment]
Today,

1. Work with SKP for index view at same place.
2. Json error resolved by SKP code check and merged in master hot fix branch,.

**Comment by VIRAG-RestAlgo** [comment]
Date 14th To 17th Oct

Work in following Task.
TaskID 248,514,517,518 Trading System.
TaskID 514,517,486,497 Algo Spinner.

Code merged in NXBlock TaskID : 248,514,517,518
Code merged in SID 514,517

**Comment by VIRAG-RestAlgo** [comment]
Working on TaskID 519

**Comment by VIRAG-RestAlgo** [comment]
TaskID 519 Done In Pronto Trading Application, and WIP in Algo Spinner

**Comment by VIRAG-RestAlgo** [comment]
TaskID 519 in algo spinner.

**Comment by VIRAG-RestAlgo** [comment]
Date 22-10-2024
TaskID 522 Done

**Comment by VIRAG-RestAlgo** [comment]
Code merge and test for BOX_IOC and algo leg control correction regarding Option type text book open in Box and BOX_IOC strategy in 2nd and 4th leg.

**Comment by VIRAG-RestAlgo** [comment]
TaskID 526 and test with Dipakbhai and meeting with Vaibhavbhai and Bhavinsir

**Comment by VIRAG-RestAlgo** [comment]
TaskID 527 + Work with dipakbhai + work in dipakbhai query + work with APN.

**Comment by VIRAG-RestAlgo** [comment]
Task ID 527 and Algo Spinner performance issue resolved..

**Comment by VIRAG-RestAlgo** [comment]
1. Spinner trade issue resolved in new deployment
2. In IOC order not add in testing so check code and find error
3. NSEFO and NSECM scriptmaster load in memory. (TaskID 528) WIP

**Comment by VIRAG-RestAlgo** [comment]
Task ID 529

**Comment by VIRAG-RestAlgo** [comment]
Trading system performance issue get in dipakbhai pc, DPR and LPPR logic merging start bt still code merging pending.

**Comment by VIRAG-RestAlgo** [comment]
Meeting with Lokeshsir regarding IOC flow in spinner and Trading System CPU usage.
Deploy and test with Dipakbhai and feedback also given to Pradeepsir.

**Comment by VIRAG-RestAlgo** [comment]
Stop loss and profit book logic dev. in algo spinner

**Comment by VIRAG-RestAlgo** [comment]
Stop loss and profit book logic dev. in algo spinner

**Comment by VIRAG-RestAlgo** [comment]
Stop loss and profit book logic dev. in algo spinner and work with Dipakbhai and pratikbhai for live issue.

**Comment by VIRAG-RestAlgo** [comment]
Stop loss and profit book logic dev. in algo spinner and trading system

**Comment by VIRAG-RestAlgo** [comment]
REV cycle will trigger in profit or stop loss identification develop and deploy

**Comment by VIRAG-RestAlgo** [comment]
Bcast count related changes and performance point of you changes in algo spinner related with stop loss and profitbooking task.

**Comment by VIRAG-RestAlgo** [comment]
TaskID 547 and 543

**Comment by VIRAG-RestAlgo** [comment]
1. M2M auto square off 
2. -1 in box strategy (5.5 hrs.)

**Comment by VIRAG-RestAlgo** [comment]
TaskID 399 Algo Spinner

**Comment by VIRAG-RestAlgo** [comment]
TaskID 399,291,560, 536 SrNo 7, 567, 560 revert

**Comment by VIRAG-RestAlgo** [comment]
TaskID 532 and work with ARV for order request code

**Comment by VIRAG-RestAlgo** [comment]
TaskID 532

**Comment by VIRAG-RestAlgo** [comment]
TaskID 532 Done and deploy. Admin, Trading and engine.

**Comment by VIRAG-RestAlgo** [comment]
03-11-2025

1. API 
2. Full Depth

**Comment by VIRAG-RestAlgo** [comment]
04-11-2025

1. API
2. Full Depth - Deploy for UAT

**Comment by VIRAG-RestAlgo** [comment]
05-11-2025

1. API

**Comment by VIRAG-RestAlgo** [comment]
Pronto API

**Comment by VIRAG-RestAlgo** [comment]
TaskID 972 wrong margin issue work with APN

**Comment by VIRAG-RestAlgo** [comment]
1. NSEFO PO
2. Live Issue in extra log (TaskID 974)
3. Live Issue mismatch qty (rejection code 16346 and record does not exist
4. TaskID 974

**Comment by VIRAG-RestAlgo** [comment]
12-11-2025

Live Issue regarding lot. (TaskID 975)

**Comment by VIRAG-RestAlgo** [comment]
13-11-2025

1. Live issue regarding pending qty TaskID 976
2. TaskID 975

**Comment by VIRAG-RestAlgo** [comment]
**14 & 15 Nov 2025**

TaskID 975

**Comment by VIRAG-RestAlgo** [comment]
17-11-2025

TaskID 970, 814, 959, 975(UAT branch merge and build)

**Comment by VIRAG-RestAlgo** [comment]
18-11-2025

API

**Comment by VIRAG-RestAlgo** [comment]
1. TaskID 814 
2. API

**Comment by VIRAG-RestAlgo** [comment]
1. TaskID 974
2. API

**Comment by VIRAG-RestAlgo** [comment]
1. TaskID 941
2. API new order and modify order

**Comment by VIRAG-RestAlgo** [comment]
1. Check and test work of SKP
2. TaskID 941 deploy for UAT
3. API
4. TaskID 980 (In engine) live issue, this is resolved and deploy for LiveUAT.

**Comment by VIRAG-RestAlgo** [comment]
1. API order request cancel and cancel all, Order book all
2. TaskID 981 - Algo Spinner  develop and Deploy for UAT.
3. TaskID 982 - Gateway simulator BSEFO develop and Deploy for UAT.

**Comment by VIRAG-RestAlgo** [comment]
TaskID : 974 (algo rule flow update)

**Comment by VIRAG-RestAlgo** [comment]
1. TaskID 974 complete and deploy for UAT now.

**Comment by VIRAG-RestAlgo** [comment]
TaskID 983 in engine, Session handling

**Comment by VIRAG-RestAlgo** [comment]
1. Admin Module TaskID 983 development and deploy for UAT.
2. Trading System TaskID 983 development and deploy for UAT.
3. RYDT Ops Engine TaskID 983 development and deploy for UAT.
4. NSEFO CTCL TaskID 983, 973 development and deploy for UAT.

**Comment by VIRAG-RestAlgo** [comment]
1. NSEFO and BSEFO Pre-Open related work.
2. NSEFO transcode 7208 structure changes by exchange so work with Support and Lokeshsir.

**Comment by VIRAG-RestAlgo** [comment]
04-12-2025 

1. Pre-Open related task.
2. Live issue regarding spinner not connect with engine
3. Live issue regarding margin in limit report while re-start engine.

**Comment by VIRAG-RestAlgo** [comment]
Today

TaskID 988 and pre-open flow done and deploy for UAT and mock for tomorrow.

**Comment by VIRAG-RestAlgo** [comment]
1. TaskID 988
2. Mock related changes in preOpen
3. TaskID 985 gateway simulator for BSEFO coding check and deploy for UAT.

**Comment by VIRAG-RestAlgo** [comment]
1. Session handling demo with Sir
2. Github task update which is in live and UAT since 30-10-2025
3. Admin pending To Do and Backlog check and discuss with ARV.
4. TaskID 931 merge and code update by me with SKP.

**Comment by VIRAG-RestAlgo** [comment]
TaskID 989 and 990 in spinner.

TaskID 989 develop and deploy for UAT

**Comment by VIRAG-RestAlgo** [comment]
TaskID 974 performance related changes done final and deploy for UAT

**Comment by VIRAG-RestAlgo** [comment]
1. Bug  sheet manage.
2. MM main API
3. pending points discuss with sir
4. IV discuss with Anilbhai and Sir

**Comment by VIRAG-RestAlgo** [comment]
Date 18,19 and 20 Dec 2025 work from Bhavnagar office.

Work with APN and Lokeshsir for RISK

**Comment by VIRAG-RestAlgo** [comment]
Today,

Work with Lokeshsir and APN TaskID 996

TaskID 996 deploy for UAT.

**Comment by VIRAG-RestAlgo** [comment]
Work in RISK with Lokeshsir and APN from 18th Dec to 24th Dec 2025

**Comment by VIRAG-RestAlgo** [comment]
TaskID 999,1000 & work with ARV for MMMain API

**Comment by VIRAG-RestAlgo** [comment]
TaskID 1001 and MMMain API SeriLog write.

**Comment by VIRAG-RestAlgo** [comment]
TaskID 1003,1004

**Comment by VIRAG-RestAlgo** [comment]
TaskID 648

**Comment by VIRAG-RestAlgo** [comment]
Work in AlgoSpinner regarding F1 buy and F2 Sell issue. (Support know point)

**Comment by VIRAG-RestAlgo** [comment]
1. TaskID 996 & 997
2. AlgoSpinner - 1000 lot in single user total 3 rule performance check and code optimized.

**Comment by VIRAG-RestAlgo** [comment]
1. TaskID 648 code merge and deploy for UAT.
2. BSEFO TBT broadcast work with Support and Lokeshsir.
3. TaskID 977 in NSEFO Adapter MD5 checksum.

**Comment by VIRAG-RestAlgo** [comment]
TaskID 974, 648, 1002 and all deploy for UAT today.

**Comment by VIRAG-RestAlgo** [comment]
1. TaskID 977, Pronto API, TaskID 1008 in Spinner, Engine and Trading

**Comment by VIRAG-RestAlgo** [comment]
1. NSECM preOpenOrder
2. Pronto API

**Comment by VIRAG-RestAlgo** [comment]
16-01-2026

work on MD5  with sir

**Comment by VIRAG-RestAlgo** [comment]
Today

RYDT API

**Comment by VIRAG-RestAlgo** [comment]
1. NSECM preOpenOrder
2. Pronto API

**Comment by VIRAG-RestAlgo** [comment]
1. MD5 testing with development for test case
2. API
3. PO order from trading with SKP

**Comment by VIRAG-RestAlgo** [comment]
1. ProntoAPI
2. PreOpenOrder
3. TaskID 1014 (Series related changes)

**Comment by VIRAG-RestAlgo** [comment]
1. ProntoAPI
2. PreOpenOrder
3. TaskID 1014 (Series related changes) - Deploy in UAT.

**Comment by VIRAG-RestAlgo** [comment]
1. ProntoAPI

**Comment by VIRAG-RestAlgo** [comment]
1. Pronto API

**Comment by VIRAG-RestAlgo** [comment]
1. Pronto API
